books/tools/soft-paper-examples.lisp

```diff
@@ -1,414 +1,3 @@
-; SOFT Examples from the ACL2-2015 Workshop paper
-;
-; Copyright (C) 2015 Kestrel Institute (http://www.kestrel.edu)
-;
-; License (an MIT license):
-;
-;   Permission is hereby granted, free of charge, to any person obtaining a
-;   copy of this software and associated documentation files (the "Software"),
-;   to deal in the Software without restriction, including without limitation
-;   the rights to use, copy, modify, merge, publish, distribute, sublicense,
-;   and/or sell copies of the Software, and to permit persons to whom the
-;   Software is furnished to do so, subject to the following conditions:
-;
-;   The above copyright notice and this permission notice shall be included in
-;   all copies or substantial portions of the Software.
-;
-;   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-;   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-;   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-;   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-;   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-;   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-;   DEALINGS IN THE SOFTWARE.
-;
-; Original author: Alessandro Coglio (coglio@kestrel.edu)
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-; This file contains the SOFT ('Second-Order Functions and Theorems') examples
-; in the ACL2-2015 Workshop paper "Second-Order Functions and Theorems in ACL2".
-
-; Comments indicate the sections and subsections of the paper.
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
 (in-package "ACL2")
 
-(include-book "soft")
```

Comment:
It's unclear whether soft should be in kestrel or a more general place.  Anyway, I'm happy to have it moved to kestrel for now.  Do you mind creating another pull request that tags these as "relocation stubs"?  We didn't have this in a well-documented place, so I added it to the wiki.
https://github.com/acl2/acl2/wiki/How-to-move-a-book

Also, your CL highlights our thus-far-unstructured desire to come up with a structure for testing outside a normal build -- I think you're pretty close to what we want already. :)